lego/apps/events/constants.py

```diff
@@ -18,6 +18,12 @@
     (EVENT, EVENT),
 )
 
+NORMAL = "normal"
+INFINITY = "infinity"
+OPEN = "open"
+TBA = "tba"
+EVENT_STATUS_TYPES = ((NORMAL, NORMAL), (INFINITY, INFINITY), (OPEN, OPEN), (TBA, TBA))
```

Comment:
Can we get some documentation somewhere that explains what the different statuses represent? I feel like it's not that self explanatory, and it would be nice to not have to look up this pr to read what they mean.